e2e/validation.go

```diff
@@ -102,5 +115,16 @@ func commonLiveVMValidators() []*LiveVMValidator {
 				"cloud-config.txt",
 			},
 		),
+		{
+			Description: "check that curl to wireserver fails",
+			Command:     "curl \"http://168.63.129.16/machine/?comp=goalstate\" -H \"x-ms-version: 2015-04-05\" -s --connect-timeout 10",
```

Comment:
the ports that should be blocked are 443 and 32526

Reply:
Got two checks for both ports now